samples/wifi/shell/prj.conf

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# WPA supplicant
+CONFIG_WPA_SUPP=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_POSIX_API=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_DNS_RESOLVER=y
+
+CONFIG_NET_STATISTICS=y
+
+CONFIG_NET_PKT_RX_COUNT=8
+CONFIG_NET_PKT_TX_COUNT=8
+
+# Below section is the primary contributor to SRAM and is currently
+# tuned for performance, but this will be revisited in the future.
+CONFIG_NET_BUF_RX_COUNT=16
+CONFIG_NET_BUF_TX_COUNT=16
+CONFIG_NET_BUF_DATA_SIZE=128
+# WPA2/3 ping
+CONFIG_HEAP_MEM_POOL_SIZE=153600
+CONFIG_NET_TC_TX_COUNT=0
+
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=4
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
+CONFIG_NET_IF_UNICAST_IPV4_ADDR_COUNT=1
+CONFIG_NET_MAX_CONTEXTS=5
+CONFIG_NET_CONTEXT_SYNC_RECV=y
+
+CONFIG_INIT_STACKS=y
+
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_SHELL=y
+
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_CONFIG_PEER_IPV6_ADDR="2001:db8::2"
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.165.100.150"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.165.100.1"
+
+CONFIG_NET_SOCKETS_POLL_MAX=4
+
+# Memories
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SHELL_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_NET_TX_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=4096
+
+# Debugging
+CONFIG_STACK_SENTINEL=y
+CONFIG_DEBUG_COREDUMP=y
+CONFIG_DEBUG_COREDUMP_BACKEND_LOGGING=y
+CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
+CONFIG_SHELL_CMDS_RESIZE=n
+#CONFIG_DEBUG=y
+CONFIG_WPA_SUPP_DEBUG_LEVEL="INFO"
+
+# Kernel options
+CONFIG_ENTROPY_GENERATOR=y
+
+# Logging
+CONFIG_LOG=y
+CONFIG_PRINTK=y
+CONFIG_SHELL=y
+CONFIG_SHELL_GETOPT=y
+CONFIG_DEVICE_SHELL=y
+CONFIG_POSIX_CLOCK=y
+CONFIG_DATE_SHELL=y
+
+# Security
+# CONFIG_CRYPTO_MBEDTLS_SHIM=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_MBEDTLS=y
```

Comment:
I think NCS is moving from mbedtls to PSA Crypto API. Is it fine to rely on mbedtls here? @frkv